src/gui/sharelinkwidget.cpp

```diff
@@ -473,6 +485,20 @@ void ShareLinkWidget::slotPushButtonCopyLinkPressed()
 #endif
 }
 
+void ShareLinkWidget::slotPushButtonMailLinkPressed()
+{
+    QString fileName = _sharePath.mid(_sharePath.lastIndexOf('/') + 1);
+
+    QDesktopServices::openUrl(QUrl(QString(
+            "mailto: "
+            "?subject=%1 has shared %2 with you"
+            "&body=%3").arg(
+            _account->credentials()->user(),
```

Comment:
@rullzer Nice catch. Since the email is coming from your personal email account and client rather than the server, the template text should be in the first person unlike other email templates.